Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuController.cs

```diff
@@ -0,0 +1,527 @@
+                {
+                    RotateTowards = GameObject.Find("Camera (eye)");
+                    if (RotateTowards == null)
+                    {
+                        Debug.LogWarning("The PanelMenuController could not automatically find an object to rotate towards.");
+                    }
+                }
+
+                if (isShown)
+                {
+                    if (RotateTowards != null)
+                    {
+                        transform.rotation = Quaternion.LookRotation((RotateTowards.transform.position - transform.position) * -1, Vector3.up);
+                    }
+                }
+
+                if (isPendingSwipeCheck)
+                {
+                    CalculateSwipeAction();
+                }
+            }
+        }
+
+        #endregion Unity Methods
+
+        #region Initialize
+
+        protected virtual void Initialize()
+        {
+            if (Application.isPlaying)
+            {
+                if (!isShown)
+                {
+                    transform.localScale = Vector3.zero;
+                }
+            }
+
+            if (controllerEvents == null)
+            {
+                transform.localPosition = new Vector3(transform.localPosition.x, transform.localPosition.y, transform.localPosition.z);
+                controllerEvents = GetComponentInParent<VRTK_ControllerEvents>();
+            }
+        }
+
+        #endregion Initialize
+
+        #region Interaction
+
+        private void BindControllerEvents()
+        {
+            controllerEvents.TouchpadPressed += new ControllerInteractionEventHandler(DoTouchpadPress);
+            controllerEvents.TouchpadReleased += new ControllerInteractionEventHandler(DoTouchpadReleased);
+            controllerEvents.TouchpadTouchStart += new ControllerInteractionEventHandler(DoTouchpadTouched);
+            controllerEvents.TouchpadTouchEnd += new ControllerInteractionEventHandler(DoTouchpadUntouched);
+            controllerEvents.TouchpadAxisChanged += new ControllerInteractionEventHandler(DoTouchpadAxisChanged);
+            controllerEvents.TriggerPressed += new ControllerInteractionEventHandler(DoTriggerPressed);
+        }
+
+        private void UnbindControllerEvents()
+        {
+            controllerEvents.TouchpadPressed -= new ControllerInteractionEventHandler(DoTouchpadPress);
+            controllerEvents.TouchpadReleased -= new ControllerInteractionEventHandler(DoTouchpadReleased);
+            controllerEvents.TouchpadTouchStart -= new ControllerInteractionEventHandler(DoTouchpadTouched);
+            controllerEvents.TouchpadTouchEnd -= new ControllerInteractionEventHandler(DoTouchpadUntouched);
+            controllerEvents.TouchpadAxisChanged -= new ControllerInteractionEventHandler(DoTouchpadAxisChanged);
+            controllerEvents.TriggerPressed -= new ControllerInteractionEventHandler(DoTriggerPressed);
+        }
+
+        public void ToggleMenu()
```

Comment:
public methods should appear first, so the order should be

* public
* protected
* private